server/models/Meeting.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+
+const meetingSchema = new mongoose.Schema({
+  user_id: {
+    type: String,
+    required: true
+  },
```

Comment:
Here we are saying any string can be a user id where we want to reference another object in the users collection. Mongoose provides a schema type for ObjectId. `mongoose.Types.ObjectId` and make sure to `ref: 'user'`. Also, JavaScript convention is camelCase and not snake_case.

Reply:
I've changed tthe type of `user_id` to `mongoose.Types.ObjectId` and added `ref: "user"`.